armory/errors.py

```diff
@@ -0,0 +1,25 @@
+import contextlib
```

Comment:
I really like the implementation of this as a context manager; nifty

I think errors.py is a confusing name for a module that imports, please change.